### PR TITLE
Fix UI drag & drop tests

### PR DESCRIPTION
**Pull request by jknaudt21**
### Description of Change

When running UI tests, I naively ran each individual test one by one instead of considering that the application state will remain after each test. This caused the drag & drop tests to fail because when the tests run in sequence, the expected coordinates of the events differ from what the tests were originally designed to check. 

### Issues Fixed

Added a "Reset" button to the view that gets clicked on each test run. The button regenerates the necessary layouts and text back to a base state so that every test is run under the same conditions.

The button is pressed at the **start** of every test run in case previous tests failed and were unable to set back the views to their original state. 

I also made the layouts slightly smaller to better fit within the screen.

### CODE REVIEW

File: src/Controls/samples/Controls.Sample.UITests/Elements/DragAndDropBetweenLayouts.xaml.cs [hunk 1]
```diff
@@ -132,5 +132,58 @@ private void OnDrop(object sender, DropEventArgs e)
 
 			AddEvent(nameof(OnDrop));
 		}
-	}
+
+		private void ResetLayouts(object sender, System.EventArgs e)
```
Comment (rmarinho) [inline src/Controls/samples/Controls.Sample.UITests/Elements/DragAndDropBetweenLayouts.xaml.cs:136]: just a nitpick but we don't use `private`

File: src/Controls/tests/UITests/Tests/DragAndDropUITests.cs [hunk 2]
```diff
@@ -51,6 +51,8 @@ public void DragAndDropBetweenLayouts()
 			App.EnterText("TargetView", "DragAndDropBetweenLayouts");
 			App.Tap("GoButton");
 
+			App.Tap("ResetButton");
```
Comment (rmarinho) [inline src/Controls/tests/UITests/Tests/DragAndDropUITests.cs:54]: I wonder if we need to "wait" that Reset finished

**Review by rmarinho — CHANGES_REQUESTED** [review]
image

Android still seems to fail, iOS and Windows are good. 

Mac seems to be hanging but not sure if its related or not

File: src/Controls/tests/UITests/Tests/DragAndDropUITests.cs (hunk 2 above)
Comment (jknaudt21) [reply src/Controls/tests/UITests/Tests/DragAndDropUITests.cs:54]: After the reset we wait for the elements to show on screen, so I think we should be alright.

**Comment by jknaudt21** [comment]
Not sure what could be causing the failures. I don't really know what could be going wrong with the Mac failures.

Regarding Android, only thing that came up was the possibility of the display being too small. I noticed that if I made the items bigger such that some are no longer visible in screen, the test fails. As such, I reduced the size of the labels. I tried running the entire sequence of tests on a Pixel 5 w API lv 30 and everything passed: 
image

**Comment by jknaudt21** [comment]
@rmarinho  iOS, Android, and Windows are passing. The other failing tests are not regarding drag & drop. Mac still hangs (and I'm not really sure its because of these changes)

**Review by rmarinho — APPROVED** [review]

**Pull request merged**